### PR TITLE
Updated test-nullspace to standards

### LINKED ISSUE #82: write test-nullspace
Update test-nullspace.C to the new testsuite standard.

### PR DESCRIPTION
**Pull request by Breush**
Closes #82.

- Checking rank + NS dim size;
- Checking the result is nullifying indeed.

### CODE REVIEW

**Comment by Breush** [comment]
@ClementPernet Updated.

File: tests/test-nullspace.C [hunk 1]
```diff
@@ -56,38 +95,40 @@ bool check(Field& F, FFLAS::FFLAS_SIDE side, size_t m, size_t n, size_t r, typen
...
+    std::cout << "NS: " << NSdim << std::endl;
+#endif
 
     if (side == FFLAS::FFLAS_SIDE::FflasRight) {
         // Right nullspace dimension + Rank == Matrix column dimension
         if (NSdim + r != n) return false;
 
+        // Ensure nullspace is full rank
+        auto NSCopy = FFLAS::fflas_new(F, n, NSdim);
+        FFLAS::fassign(F, n, NSdim, NS, NSdim, NSCopy, NSdim);
```
Comment (ClementPernet) [inline tests/test-nullspace.C]: FYI: there is an `FFPACK::IsSingular` method!

File: tests/test-nullspace.C [hunk 2]
```diff
@@ -157,24 +192,21 @@ int main(int argc, char** argv)
     }
 
     srand(seed);
+    std::cout << "With seed: " << seed << std::endl;
```
Comment (ClementPernet) [inline tests/test-nullspace.C]: Not sure I like to display the seed here only: when running in loop mode, it is relevant to display the seed generating the last iteration that caused the failure. Maybe, print at the endd `seed-1` ?

File: tests/test-nullspace.C (hunk 1 above)
Comment (Breush) [reply tests/test-nullspace.C]: `FFPACK::IsSingular` doesn't check for full rank of non-square. See #175

**Pull request merged**